apps/web/components/Navbar.tsx

```diff
@@ -83,7 +83,7 @@ export default function NavBar() {
           <Navigation>
             <Link href="#">
               <ListRoutes>
-                <MdHelpCenter fontSize={19} />
+                <MdHelpCenter fontSize={17} />
```

Comment:
@MFelipeSilva why is this icon a different size than the others?

Reply:
because with 1 size more, it stays the same size as the others

Reply:
@MFelipeSilva ok, but is better to use the same size in all icons.